src/cmdc/datasets/uscensus/data.py

```diff
@@ -0,0 +1,198 @@
+import io
+import json
+import pandas as pd
+import random
+import requests
+import sqlalchemy as sa
+
+from .census import ACS
+from ..base import OnConflictNothingBase
+from ..db_util import TempTable
+
+uscensus_variable_insert = """
+"""
+
+class ACS_CMDC(ACS, OnConflictNothingBase):
```

Comment:
No underscores in class name.

I propose we do one of two options: 

1. rename this to `ACSScraper`
2. rename the other ACS class to `ACSBase` or `ACSAPI`

FWIW I prefer option 2